specs/init.md

```diff
@@ -33,13 +33,31 @@ it's own service, make sure it's always running by re-spawning (if needed), a `o
 - All services logs are written directly to kernel ring-buffer
   - Optionally later on, one of the daemon can be responsible of reading the logs and push them somewhere else.
 - once a service update it's status, other `waiting` threads (that depends on this one) will get freed to take start.
-## Controlling:
+
+## Controlling
 A unix socket interface can be used to control the init process as follows
 - Shutdown, Reboot:
   - the manager, will set global runlevel to shutdown, ask individual services to die.
   - once each service exits, their monitor threads will not re-spawn due to global runtime state
   - once all services are down, a shutdown (or reboot) is performed.
 - Status inquiry
   - List all configured services and their status.  
+- Sync to config
+  - A new service can be added to the configurations directory, then the init will be asked to re-scan, new service MUST be scheduled to run, deleted service MUST be stopped. Changed services
+  are not going to re-spawn. 
+- Reload
+  - Reload is given a specific service name, to reload the config in case the parameter or the env list has changed. Reload is NOT done automatically with `sync`. Must be explicitly called with certain services. 
+
+# Configuration
+Each service must have a configuration file that defines how a service should start.
 
-> Do we need to support starting and stopping individual services ? with start/stop commands ?
\ No newline at end of file
+```yaml
+exec: full command line goes here
+oneshot: true/false # default to false
```

Comment:
I think we need to think about other restart policy.
We have learned that just plainly restarting a failing service contently can end up in a pretty heavy loop that will consume a lot of resources for nothing.
Something that let us choose to try to restart the service but with a increasing pause between retries ...

Reply:
Yeah, a backoff strategy will be implemented to avoid getting stuck in a busy loop.